src/Sylius/Bundle/AdminBundle/templates/product_review/form/sections/info/status.html.twig

```diff
@@ -0,0 +1,20 @@
+{% from '@SyliusAdmin/shared/helper/icon.html.twig' import icon %}
+
+{% set status = hookable_metadata.context.resource.status %}
+{% set label = 'sylius.ui.' ~ status %}
+
+{% set status_badge = {
+    accepted: 'success',
+    new: 'secondary',
+    rejected: 'warning',
+} %}
+
+
+<div class="card mb-3">
```

Comment:
```suggestion
} %}

<div class="card mb-3">
```

Reply:
Fixed in https://github.com/Sylius/Sylius/pull/16325